src/inputs/labeled-inputs.scss

```diff
@@ -52,26 +53,36 @@
     font-size: $iui-font-size-small;
     line-height: $iui-line-height;
     margin-top: $iui-inputs-margin;
-
     @include themed {
       color: t(iui-text-color-muted);
     }
 
-    svg {
+    > svg,
+    :where(:not(.iui-button)) svg {
```

Comment:
I don't think `:where` is needed here.

Reply:
Replaced with `:is(:not(.iui-button)) svg`, credit goes to @mayank99 for finding this one.